pkgs/tools/misc/github-copilot-cli/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildNpmPackage, fetchzip }:
+
+buildNpmPackage rec {
+  pname = "github-copilot-cli";
+  version = "0.1.33";
+
+  src = fetchzip {
+    url = "https://registry.npmjs.org/@githubnext/${pname}/-/${pname}-${version}.tgz";
+    hash = "sha256-uTv6Z/AzvINinMiIfaaqRZDCmsAQ7tOE5SpuecpzGug=";
+  };
+
+  npmDepsHash = "sha256-VIg9a63GH246SbmK4Q8CwA2jdaaOwNUXoJkuDVwy5jE=";
+
+  postPatch = ''
+    cp ${./package-lock.json} package-lock.json
+  '';
+
+  dontNpmBuild = true;
+
+  meta = with lib; {
+    description = "A CLI experience for letting GitHub Copilot help you on the command line";
+    homepage = "https://githubnext.com/projects/copilot-cli/";
+    license = licenses.free;
```

Comment:
Proof?

Reply:
This was an oversight on my part. Sorry about that. The package on NPM doesn't contain a license, and not knowing much about how licenses work, I didn't know what to select for the license, so I picked `licenses.free`, and hoped that other folks would point out if that was wrong. Might be a good idea for the `nixpkgs` manual to explicitly say in the [`lib.licenses.unfree`](https://nixos.org/manual/nixpkgs/stable/#lib.licenses.unfree-unfree) that if a package does not have a license explicitly specified, the the package should be marked as `unfree`.